### PR TITLE
Unit test for Sparkline Chart using Rewire

### PR DESCRIPTION
**Pull request by srmukher**
Unit test for Sparkline Chart using Rewire.

### CODE REVIEW

File: tools/test/Sparkline/SparklineBase.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+"use strict";
```
Comment (AtishayMsft) [inline tools/test/Sparkline/SparklineBase.js:1]: remove this file

File: package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/fluentui-charting-contrib"
+    },
+    "license": "MIT",
+    "scripts": {
+        "test": "jest"
+    },
+    "devDependencies": {
+        "jest": "29.7.0",
```
Comment (AtishayMsft) [inline package.json:14]: import just react-charting, rewire, RTL, jest etc

File: tools/test/Sparkline/SparklineBase.js (hunk 1 above)
Comment (srmukher) [reply tools/test/Sparkline/SparklineBase.js:1]: But this is the one which is required for testing

File: package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@fluentui/fluentui-charting-contrib",
```
Comment (AtishayMsft) [inline package.json:2]: move this package.json inside tools/UnitTestApp

File: package.json (hunk 2 above)
Comment (srmukher) [reply package.json:14]: yes, rest are required as they are part of the SparklineBase.js.

File: tools/test/Sparkline/SparklineUT.test.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
+const sparkline1Points = {
+  chartTitle: '10.21',
+  lineChartData: [
+    {
```
Comment (AtishayMsft) [inline tools/test/Sparkline/SparklineUT.test.tsx:4]: put this file inside tools/UnitTestApp

Comment (srmukher) [reply tools/test/Sparkline/SparklineUT.test.tsx]: done

File: package.json (hunk 3 above)
Comment (srmukher) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (srmukher) [reply package.json]: done

File: tools/test/Sparkline/SparklineBase.js (hunk 1 above)
Comment (srmukher) [reply tools/test/Sparkline/SparklineBase.js]: done

File: tools/UnitTestApp/package.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@fluentui/fluentui-charting-contrib",
+    "version": "1.0",
+    "description": "Experimental React charting Contribution environment for building experiences for React charting library.",
```
Comment (AtishayMsft) [inline tools/UnitTestApp/package.json:4]: update name and description of package.json

Comment (srmukher) [reply tools/UnitTestApp/package.json]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**